pull.py

```diff
@@ -47,6 +47,20 @@
         'CC-BY-ND-3.0',
         'CC-BY-ND-4.0',
     ],
+    'CC-BY': [ # any version
```

Comment:
I don't see any `CC-BY` slug on their page.  This link should not take you to any specific entry: https://www.gnu.org/licenses/license-list.html#CC-BY

Reply:
I've added checks in `master`'s `pull.py` to watch out for this sort of thing.

Reply:
I just pushed a correction for this and CC-BY-SA to the pull request branch